modular_skyrat/modules/deforest_medical_items/code/injectors.dm

```diff
@@ -0,0 +1,208 @@
+	base_icon_state = "calopine"
+	icon_state = "calopine"
+	list_reagents = list(
+		/datum/reagent/medicine/atropine = 10,
+		/datum/reagent/medicine/coagulant/fabricated = 5,
+		/datum/reagent/medicine/salbutamol = 5,
+		/datum/reagent/toxin/staminatoxin = 5,
+	)
+
+// Coagulant, really not a whole lot more
+/obj/item/reagent_containers/hypospray/medipen/deforest/coagulants
+	name = "coagulant-S injector"
+	desc = "A Deforest branded autoinjector, loaded with a mix of coagulants to prevent and stop bleeding."
+	base_icon_state = "coagulant"
+	icon_state = "coagulant"
+	list_reagents = list(
+		/datum/reagent/medicine/coagulant = 5,
+		/datum/reagent/medicine/salglu_solution = 15,
+		/datum/reagent/impurity = 5,
+	)
+
+// Stimulant centered around ondansetron
+/obj/item/reagent_containers/hypospray/medipen/deforest/krotozine
+	name = "krotozine manipulative stimulant injector"
+	desc = "A Deforest branded autoinjector, loaded with a mix of stimulants of weak healing agents."
+	base_icon_state = "krotozine"
+	icon_state = "krotozine"
+	list_reagents = list(
+		/datum/reagent/medicine/ondansetron = 5,
+		/datum/reagent/drug/kronkaine = 5,
+		/datum/reagent/medicine/omnizine/protozine = 10,
+		/datum/reagent/drug/maint/tar = 5,
+	)
+
+// Stuff really good at healing burn stuff and stabilizing temps
+/obj/item/reagent_containers/hypospray/medipen/deforest/lepoturi
+	name = "lepoturi burn treatment injector"
+	desc = "A Deforest branded autoinjector, loaded with a mix of medicines to rapidly treat burns."
+	base_icon_state = "lepoturi"
+	icon_state = "lepoturi"
+	list_reagents = list(
+		/datum/reagent/medicine/mine_salve = 5,
+		/datum/reagent/medicine/leporazine = 5,
+		/datum/reagent/medicine/c2/lenturi = 10,
+		/datum/reagent/toxin/staminatoxin = 5,
+	)
+
+// Stabilizes a lot of stats like drowsiness, sanity, dizziness, so on
+/obj/item/reagent_containers/hypospray/medipen/deforest/psifinil
+	name = "psifinil personal recovery injector"
+	desc = "A Deforest branded autoinjector, loaded with a mix of medicines to remedy many common ailments, such as drowsiness, pain, instability, the like."
+	base_icon_state = "psifinil"
+	icon_state = "psifinil"
+	list_reagents = list(
+		/datum/reagent/medicine/modafinil = 10,
+		/datum/reagent/medicine/psicodine = 10,
+		/datum/reagent/medicine/leporazine = 5,
+	)
+
+// Helps with liver failure and some drugs, also alcohol
+/obj/item/reagent_containers/hypospray/medipen/deforest/halobinin
+	name = "halobinin soberant injector"
+	desc = "A Deforest branded autoinjector, loaded with a mix of medicines to remedy the effects of liver failure and common drugs."
+	base_icon_state = "halobinin"
+	icon_state = "halobinin"
+	list_reagents = list(
+		/datum/reagent/medicine/haloperidol = 5,
+		/datum/reagent/medicine/antihol = 5,
+		/datum/reagent/medicine/higadrite = 5,
+		/datum/reagent/medicine/silibinin = 5,
+		/datum/reagent/nitrous_oxide = 5,
```

Comment:
Again, this one's kinda bad and will just end up making bleeds worse, not sure that's a good idea.